src/DssSpell.t.sol

```diff
@@ -179,8 +201,8 @@ contract DssSpellTest is DSTest, DSMath {
             vow_bump:              10000,                   // In whole Dai units
             vow_hump:              4 * MILLION,             // In whole Dai units
             cat_box:               15 * MILLION,            // In whole Dai units
-            osm_mom_authority:     address(0),              // OsmMom authority
-            flipper_mom_authority: address(0),              // FlipperMom authority
+            osm_mom_authority:     address(newChief),       // OsmMom authority
```

Comment:
Don't forget to test for the `pause_authority` here

Reply:
Good catch! added.